packages/typescript/internal/ts_project.bzl

```diff
@@ -665,11 +665,7 @@ def ts_project_macro(
         nodejs_binary(
             name = tsc_worker,
             data = [
-                # BEGIN-INTERNAL
-                # Users get this dependency transitively from @bazel/typescript
-                # but that's our own code, so we don't.
-                "@npm//protobufjs",
-                # END-INTERNAL
+                Label("@npm//protobufjs"),
```

Comment:
this change seems wrong, we don't know that the user depends on protobufjs, and the package manager might not hoist it. It might appear in `@npm//@bazel/typescript:node_modules/protobufjs`
does your fix still work without this change?

Reply:
@alexeagle during package publish it will be changed into `//@bazel/typescript` (https://github.com/bazelbuild/rules_nodejs/pull/2652/files#diff-8da1b34bdafc2b3437699abe4585377bdc6e9e72d4158b47f9c4a3e22262238bR36). I did that change by the analogy of what already has been here. Is there any other good way to solve this problem?